client/src/components/modal.tsx

```diff
@@ -40,122 +51,126 @@ const Modal: React.FC = () => {
                     location: '',
                     notes: '',
                 });
-                setErrors('');
+                setErrors({
```

Comment:
is this the actually tabbing? or is this just github being weird

Reply:
think I fixed...more VSCode tabbing fun!